action.yml

```diff
@@ -206,6 +206,16 @@ outputs:
   run-id:
     description: >
       The id of the one time job run for the notebook.
+  add-pr-comment:
+    description: >
+      A boolean flag for whether to comment to the PR that triggered this action with the Notebook result.
+    default: 'false'
+    required: false
+  pr-github-token:
+    description: >
+      The Github Token for the repo that will be used to add a comment to the PR.
```

Comment:
Should we add a little to these docs - e.g. could the caller just pass `GITHUB_TOKEN` here / should we encourage that? Or does that token not have enough permissions to leave PR comments?

Reply:
I wonder if instead of a boolean + token, we should just add a `pr-comment-github-token` arg that if specified, enables PR comments using the specified token

Reply:
That makes sense let me reformat